sphinx/source/docs/user_guide/server.rst

```diff
@@ -818,57 +810,43 @@ embeds the sliders app, but changes the plot title *before* passing to the user:
         app.run(port=8080)
 
 .. warning::
-    It is possible to use ``bokeh.client`` to build up apps "from scratch",
-    outside a Bokeh server, including running and servicing callbacks by making
-    a blocking call to ``session.loop_until_closed`` in the external Python
-    process using ``bokeh.client``. This usage has a number of inherent
-    technical disadvantages and should be considered unsupported.
+    You can use ``bokeh.client`` to build apps "from scratch", outside the
+    Bokeh server, including running and servicing callbacks by making a
+    blocking call to ``session.loop_until_closed`` in the external Python
+    process. However, this usage has a number of inherent technical
+    disadvantages and is not supported.
```

Comment:
`session.loop_until_closed` has already been removed in fact, I think this entire warning can be deleted.